src/test/java/net/minecraftforge/debug/world/ForgeChunkManagerTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Minecraft Forge
- * Copyright (c) 2016-2021.
+ * Copyright (c) 2016-2020.
```

Comment:
Why are you changing the copyright?

Reply:
Again from my [previous comment](https://github.com/MinecraftForge/MinecraftForge/pull/7655/commits/13d20fd3e44ef2b575663bd455ff39898b0ad905#r580661234), this is the work of the `updateLicenses` task.